AirportTicketBookingSystem/Domain/Services/FlightService.cs

```diff
@@ -55,6 +55,6 @@ public double GetPriceByClass(Flight flight, SeatClass seatClass)
         {
             return price;
         }
-        return flight.Prices.ContainsKey("c") ? flight.Prices["economy"] : 0; 
+        return flight.Prices.ContainsKey("economy") ? flight.Prices["economy"] : 0; 
```

Comment:
we shouldn't hardcode strings, are these keys fixed? if we know the keys, we can save them as constants in a separate  class. Lets update the other areas of code where we hardcode the key as well.
Actually i noticed you have an enum with the seat values, maybe use that instead

Reply:
and if economy is the default setting, then lets save in this service a default constant called DEFAULT_SEAT with economy value, so we know why we use economy here.